validator/sawtooth_validator/gossip/gossip.py

```diff
@@ -474,6 +499,25 @@ def stop(self):
                 # Connection has already been disconnected.
                 pass
 
+    def _get_peered_connections(self):
+        peers = self._gossip.get_peers()
+
+        return [conn_id for conn_id in peers
+                if self._connection_statuses[conn_id] == PeerStatus.PEER]
+
+    def _request_chain_head(self, peered_connections):
+        """Request chain head from the given peer ids.
+
+        Args:
+            peered_connecions (:list:str): a list of peer connection ids where
+                the requests will be sent.
+        """
+        if not peered_connections:
```

Comment:
This check is redundant, both because it's already checked in `run` and because iterating over an empty is fine.